service-worker/bootstrap.js

```diff
@@ -249,6 +243,20 @@ self.addEventListener('install', event => {
     })
 })
 
+self.addEventListener('fetch', event => {
+  // Catches all non-prefetch requests and aborts in-progress prefetches
+  // until the request finishes, then resumes prefetching
+  abortPrefetches()
+  event.respondWith(
+    (async function() {
```

Comment:
Don't need async here. Also it would be nice to call resume, only if there is something to resume, otherwise it pollutes console. For example something like that: 
`
  event.respondWith(
    fetch(event.request).then(resp => {
      if (toResume.size > 0) {
        resumePrefetches()
      }
      return resp
    }),
  )
`